ethicalads-theme/templates/ea/homepage.html

```diff
@@ -76,7 +75,7 @@ <h3>Reach developers</h3>
         <h3>Focused and relevant</h3>
 
         <!-- Text -->
-        <p class="text-gray-800 mb-6 mb-md-0">We believe ads can <strong>complement content</strong>. Every advertiser and publisher is hand approved to ensure that every ad is <strong>relevant and interesting</strong> and our audience remains developer-focused.</p>
+        <p class="text-gray-800 mb-6 mb-md-0">We believe ads can <strong>complement content</strong>. Every advertiser and publisher is hand approved to ensure that every ad is <strong>relevant and interesting</strong>.</p>
```

Comment:
`We believe` feels wrong here. Having 2 bolded things also feels a bit off. How about:

```suggestion
        <p class="text-gray-800 mb-6 mb-md-0">Each advertiser and publisher is hand-approved so that every ad is <strong>relevant and interesting</strong>.</p>
```